src/Extensions/ElementalPageExtension.php

```diff
@@ -9,6 +9,10 @@
 use SilverStripe\Core\Config\Config;
 use SilverStripe\View\SSViewer;
 
+/**
+ * @method ElementalArea ElementalArea
+ * @property ElementalArea ElementalArea
```

Comment:
I don't think it's a property, is it?

Reply:
Unary components are also accessible as properties on `DataObject`s 💥

Reply:
TIL 👍